### PR TITLE
10466-Reskin-demo_app

### PR DESCRIPTION
**Pull request by dominicmarmionswrve**
https://swrvedev.jira.com/browse/SWRVE-10466

@Sergio-Mira @shanetmoore Reskin of Demo app. Can had more features and functionality at later date.

### CODE REVIEW

File: SwrveSDKDemo/src/main/java/com/swrve/sdk/demo/ConfigContentFragment.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+        String apikey = SwrveSDK.getApiKey();
+        TextView textViewApikey = (TextView) getView().findViewById(R.id.config_apikey);
+        textViewApikey.setText(apikey);
+
+        String userId = SwrveSDK.getUserId();
+        TextView textViewUserId = (TextView) getView().findViewById(R.id.config_userid);
+        textViewUserId.setText(userId);
+
+        //SwrveSDK.get
```
Comment (Sergio-Mira) [inline SwrveSDKDemo/src/main/java/com/swrve/sdk/demo/ConfigContentFragment.java]: Remove

File: SwrveSDKDemo/src/main/java/com/swrve/sdk/demo/MainActivity.java [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+    }
+
+    @Override
+    protected void onResume() {
+        super.onResume();
+        SwrveSDK.onResume(this);
+    }
+
+    @Override
+    public void onLowMemory() {
```
Comment (Sergio-Mira) [inline SwrveSDKDemo/src/main/java/com/swrve/sdk/demo/MainActivity.java]: Could you add 'onNewIntent' too?

**Comment by Sergio-Mira** [comment]
:rocket:

File: SwrveSDKDemo/src/main/java/com/swrve/sdk/demo/MainActivity.java (hunk 2 above)
Comment (dominicmarmionswrve) [inline SwrveSDKDemo/src/main/java/com/swrve/sdk/demo/MainActivity.java]: Do we have a SwrveSDK.onNewIntent? I don't see one......

Comment (Sergio-Mira) [inline SwrveSDKDemo/src/main/java/com/swrve/sdk/demo/MainActivity.java]: Maybe we should, we have processIntent(intent) instead

Comment (dominicmarmionswrve) [inline SwrveSDKDemo/src/main/java/com/swrve/sdk/demo/MainActivity.java]: OK, will have a think about that one and add as a new jira if we think we'll do it.

**Pull request merged**